src/backend/Dockerfile.go-api

```diff
@@ -0,0 +1,15 @@
+FROM golang
+
+RUN go get -u github.com/kardianos/govendor
+RUN go install github.com/kardianos/govendor
+
+ADD . /go/src/github.com/CaliOpen/CaliOpen/src/backend
+WORKDIR /go/src/github.com/CaliOpen/CaliOpen/src/backend
+
+RUN govendor sync -v
```

Comment:
to be sure to use the govendor bin inside the docker I'll suggest `$GOPATH/bin/govendor`